rasa/core/domain.py

```diff
@@ -652,10 +673,17 @@ def get_parsing_states(self, tracker: "DialogueStateTracker") -> Dict[Text, floa
         # Set all set slots with the featurization of the stored value
         for key, slot in tracker.slots.items():
             if slot is not None:
-                for i, slot_value in enumerate(slot.as_feature()):
-                    if slot_value != 0:
-                        slot_id = f"slot_{key}_{i}"
-                        state_dict[slot_id] = slot_value
+                if slot.value == "None" and slot.as_feature():
```

Comment:
@Ghostvv Do we an issue to fix this?

Reply:
do you mean the issue in rasa repo? no. But it should be a part of RulePolicy refactor

Reply:
added it to the issue